### PR TITLE
change cmo to remoteWrite directly, using oauth2

### PR DESCRIPTION
**Pull request by RaphaelBut**
### What type of PR is this?
cleanup

### What this PR does / why we need it?
We are removing the token-refresher by using prometheus remoteWrite directly.
The secret, previously used by token-refresher, is read and used by prometheus now to do oauth2.
Removal of the tokenrefresher resources will be done in a follow up PR, once this has been validated in stage.

### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/OSD-18526 

_Fixes #_

### Pre-checks (if applicable):
- [X ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

File: resources/cluster-monitoring-config/config.yaml
Comment (mjlshen) [inline resources/cluster-monitoring-config/config.yaml:1]: You'll also need to add `OBSERVATORIUM_URL` as a parameter to scripts/template.yml

Comment (RaphaelBut) [reply resources/cluster-monitoring-config/config.yaml:1]: I added it via the app interface deployment to be env specific here https://gitlab.cee.redhat.com/service/app-interface/-/commit/421fdf8af0ae7db768fb6340dedbc9a0305e599e 

This was previously set via the secret as well, but the cmo lacks the ability to read it .

Comment (RaphaelBut) [reply resources/cluster-monitoring-config/config.yaml:1]: Ah it needs to be added here as well https://github.com/openshift/managed-cluster-config/blob/master/scripts/templates/template.yaml#L4-L30

**Review by mjlshen — APPROVED** [review]
/lgtm

**Pull request merged**